src/main/java/decodes/cwms/CwmsTimeSeriesDb.java

```diff
@@ -782,7 +786,20 @@ public CwmsTimeSeriesDb()
 		curTimeName = "sysdate";
 		maxCompRetryTimeFrmt = "%d*1/24";
 		module = "CwmsTimeSeriesDb";
-		
+		cacheMap.put(Screening.class,
```

Comment:
For this particular one I know this screening only works in CWMS at the moment and the test is simply to open the screening editor and work, plus have a screening comp.

I'll be setting up the HEC T7 to use this branch for one of the datasets, probably SPK since no one is using it.

Otherwise the other cache operations, except unfortunately the reload, are covered under the integration tests.